src/seedu/addressbook/data/AddressBook.java

```diff
@@ -74,4 +78,16 @@ public boolean equals(Object other) {
                 || (other instanceof AddressBook // instanceof handles nulls
                         && this.allPersons.equals(((AddressBook) other).allPersons));
     }
+
+    public List<ReadOnlyPerson> sortPersons( List<ReadOnlyPerson> allPersons) {
+        if (allPersons.size()>1) {
```

Comment:
There should be a space between `allPerson.size()` ,`>` and `1`

Refer to [CS2113T Java Coding Standard](https://nuscs2113-ay1819s2.github.io/website/coding-standards/java/intermediate.html) - White space within a statement section.